src/app/workspace/resource/representation/still-image/still-image.component.ts

```diff
@@ -684,11 +677,34 @@ export class StillImageComponent implements OnChanges, OnDestroy {
         loc.y = loc.y * aspectRatio;
 
         this._viewer.addOverlay({
-            element: elt,
-            location: loc
+            element: regEle,
+            location: loc,
+        });
+
+        const tracker = new OpenSeadragon.MouseTracker({
```

Comment:
I would add a comment explaining why this is necessary since I can see someone deleting this is the future because it doesn't appear to do anything, especially since the function is empty

Reply:
ah yes, we talked about.
Added in 6a830998